test/routes/ips.js

```diff
@@ -26,15 +26,15 @@ describe('/ips', function () {
     var res = {}
     var req = {
       forwarded: {
-        ips: [
-          '0.0.0.0',
-          '1.1.1.1'
-        ]
+        for: {
```

Comment:
This is the only change that I think we need to undo in a future commit, for now this should be fine.